eth/integrity/snap_blocks_read.go

```diff
@@ -10,20 +10,25 @@ import (
 	"github.com/ledgerwatch/log/v3"
 )
 
-func SnapBlocksRead(db kv.RoDB, blockReader services.FullBlockReader, ctx context.Context, failFast bool) error {
+func SnapBlocksRead(ctx context.Context, db kv.RoDB, blockReader services.FullBlockReader, from, to uint64, failFast bool) error {
 	defer log.Info("[integrity] SnapBlocksRead: done")
 	logEvery := time.NewTicker(10 * time.Second)
 	defer logEvery.Stop()
 
 	maxBlockNum := blockReader.Snapshots().SegmentsMax()
-	for i := uint64(0); i < maxBlockNum; i += 10_000 {
+
+	if to != 0 && maxBlockNum > to {
+		maxBlockNum = 2
```

Comment:
out of curiosity - why is this necessary? is it worth leaving a comment?

Reply:
hmm - should say: maxBlockNum = to

Reply:
@mh0lt this typo still needs fixing